.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: dart
+with_content_shell: true
+script: ./tool/travis.sh
```

Comment:
You can directly put `pub run test ....` here and skip `tools/travis.sh`. See https://gist.github.com/a14n/441101c997ee8f935d9c